Lesson5/Task1/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Lesson5
+{
+    class Program
+    {
+        static int Check()
+        {
+            int checkedInput = 0;
+            string input = Console.ReadLine();
+            while (!Int32.TryParse(input, out checkedInput))
+            {
+                Console.Write("Integer required! Try again: ");
+                input = Console.ReadLine();
+            }
+
+            return checkedInput;
+        }
+
+        static void Main()
+        {
+            Rectangle myRectangle = new Rectangle();
```

Comment:
@okuzmina You could create your own constructor with 2 params, getting 2 sides, to not fill every side particularly with correct values, after the rectangle object is already created